build.rs

```diff
@@ -20,6 +20,8 @@ fn dump_flags(cache_dir: &Path, version: &str) -> Vec<u8> {
         };
         let result = bazelisk_cmd
             .env("USE_BAZEL_VERSION", version)
+            .arg("--nosystem_rc")
+            .arg("--nohome_rc")
```

Comment:
in `bazel_flags.rs`, we are instead using `--ignore_all_rc_files`. I guess we should use that one instead, since we want **_no_** bazelrc files to influence this command

Reply:
fixed